server/README.md

```diff
@@ -0,0 +1,39 @@
+# Dose Amigos Server Side Code
```

Comment:
It doesn't really hurt anything, but the server folder is unnecessary -- all of this repo is server-side code.

The ionic2 code will be in a separate dose-amigos-ionic repo.